content/docs/compute/custom-image.md

```diff
@@ -0,0 +1,196 @@
+```
+
+:::note
+
+- The name is modified and prefixed with `cust-` once created to distinguish custom images from system images provided by Civo .
+- The `civo diskimage create` command does **both** the image record creation **and** the file upload in one step—you don’t need to call a separate “upload” command.
+
+:::
+By default, the image is created in your currently-selected region.
+
+You can see all available options for disk image creation on the CLI by running `civo diskimage create --help`.
+
+</TabItem>
+</Tabs>
+
+### Available parameters for creation
+
+| Parameter                 | Description                                                          |
+| ------------------------- | -------------------------------------------------------------------- |
+| `--name`                  | Name of the disk image (required)                                    |
+| `--distribution`          | OS distribution (e.g., "ubuntu", "centos") (required)                |
+| `--version`               | Version of the operating system (required)                           |
+| `--path`                  | Path to the disk image file (.raw or .qcow2 , max 600 GB) (required) |
+| `--os` (optional)         | Operating system type (defaults to "linux", can be "windows")        |
+| `--logo_path` (optional ) | Path to your logo file (SVG format, max 5 MB)                        |
+
+## Checking disk image status
+
+<Tabs groupId="check-diskimage">
+<TabItem value="cli" label="Civo CLI">
+
+After uploading an image, it goes through several processing stages. You can check the status of your disk image with:
+
+```console
+$ civo diskimage show cust-ubuntuimage-148ee-wrodm
+                ID : a3721853-f976-49fd-83e4-1b7f48ecdb58
+              Name : cust-ubuntuimage-148ee-wrodm
+      Distribution : ubuntu
+           Version : 22.04
+                OS : linux
+             State : available
+         Image URL : https://objectstore.staging.civo.com/disk-images/my-ubuntu-image/disk.raw?X-Amz-Algorithm=AWS4-HMAC-SHA256&SignedHeaders=host&x-id=PutObject&X-Amz-Signature=8a7d32f...
+          Logo URL : https://objectstore.staging.civo.com/disk-images/my-ubuntu-image/logo.svg
+       Image Size : 2.5 GB
+```
+
+</TabItem>
+</Tabs>
+
+### Understanding disk image states
+
+The disk image states include:
+
+- `upload_pending`: Initial state after creation, awaiting upload
+- `upload_completed`: Upload finished, MD5 checksum verified
+- `pending`: Image is being processed for distribution
+- `available`: Image is ready to use for creating instances
+
+- `upload_expired`: Upload not completed within 200 minutes (images in this state **cannot** be used to create instances; delete and retry)
+
+- `error`: Issue with upload or processing
+
+:::note
+Only images in the **available** state can be used to create instances.
+:::
+
+## Listing custom disk images
+
+<Tabs groupId="list-diskimage">
+<TabItem value="cli" label="Civo CLI">
+
+To see all your custom disk images:
```

Comment:
Would this be worth to have a note that they are region-specific and a region parameter is accepted?